examples/deform_geometry/runScript.py

```diff
@@ -0,0 +1,161 @@
+"""
+This script demonstrates the deformation of a geometry object using FFD and
+the process for exporting the geometry as a tecplot or IGES file.
+"""
+from pygeo import pyGeo, DVGeometry
+import numpy as np
+import argparse
+
+def deform_liftingsurface():
+    # =========================================================================
+    # Set Up pyGeo Object
+    # =========================================================================
+    # rst LiftingSurface
+    # ---------------------  Lifting Surface Definition --------------------- #
+    # Airfoil file
+    naf = 10  # number of airfoils
+    airfoil_list = ["./geo/rae2822.dat"] * naf
+
+    # Airfoil leading edge positions
+    x = np.linspace(0.0, 7.5, naf)
+    y = np.linspace(0.0, 0.0, naf)
+    z = np.linspace(0.0, 14.0, naf)
+
+    offset = np.zeros((naf, 2))  # x-y offset applied to airfoil position before scaling
+
+    # Airfoil rotations
+    rot_x = [0.0] * naf
+    rot_y = [0.0] * naf
+    rot_z = [0.0] * naf
+
+    # Airfoil scaling
+    chord = np.linspace(5.0, 1.5, naf)
+
+    # Run pyGeo
+    geo = pyGeo(
+        "liftingSurface",
+        xsections=airfoil_list,
+        scale=chord,
+        offset=offset,
+        x=x,
+        y=y,
+        z=z,
+        rotX=rot_x,
+        rotY=rot_y,
+        rotZ=rot_z,
+        tip="rounded",
+        bluntTe=True,
+        squareTeTip=True,
+        teHeight=0.25 * 0.0254,
+    )
+    # rst LiftingSurface (end)
+
+    # =========================================================================
+    # Setup DVGeometry object
+    # =========================================================================
```

Comment:
Is the dvgeo part identical across these scripts? if so, we can just have another smaller function that creates the DVGeo object and just returns it? Less code duplication that way I think?

Reply:
Yes, they are the same. I thought about combining them but figured it would be easier for a user to see the complete example. I see your point though, I'll convert it into a function to avoid repeating those lines.

Reply:
The way I had this in mind was that we would have a function that would return the built DVGeo object, but I think your version might be easier to follow since the flow is linear. So I think this looks good!